connector/src/lib.rs

```diff
@@ -100,10 +100,15 @@ pub struct SnapshotSourceConfig {
     pub rpc_http_url: String,
 }
 
+#[derive(Clone, Debug, Deserialize)]
+pub enum EntityFilter {
+    FilterByAccountIds(Vec<String>),
+    FilterByProgramId(String),
```

Comment:
not a required change since it was like this before, but it'd make more sense to me if this used `Pubkey` over `String`.

Reply:
will go for Pubkey -  thought the same